server/api/orders.js

```diff
@@ -2,26 +2,19 @@ const router = require('express').Router()
 const Order = require('../db').models.order;
 const Order_Product = require('../db').models.order_product;
 
-//testRoute
-router.get('/:id/totalQuant', (req, res, next) => {
-  Order.findById(req.params.id)
-    .then(function(order){
-      return order.totalQuantity(req.params.id)
-    })
-    .then(totalQuant => {
-      res.json(totalQuant)
-    })
 
-})
-
-router.get('/:id/totalCost', (req, res, next) => {
-  Order.findById(req.params.id)
-    .then(function(order){
-      return order.totalCost(req.params.id)
+router.post('/', (req, res, next) => {
+  if (!req.session.orderId) {
+    Order.create({
+      status: 'pending'
     })
-    .then(totalCost => {
-      res.json(totalCost)
+    .then(order => {
+      req.session.orderId = order.id; 
+      order.setProduct(req.body.product);
```

Comment:
do we also need to set the product amount/quant on the join table here?

Reply:
@cylau1031 those will likely be separate calls, this is simply to create an order and assign the session.orderId - they're separate so that those add quantity functions can be modular and reusable